### PR TITLE
Bcreane bump libcalico go

### PR DESCRIPTION
**Pull request by caseydavenport**
## Description
A few sentences describing the overall goals of the pull request's commits.
Please include
- the type of fix - (e.g. bug fix, new feature, documentation)
- some details on _why_ this PR should be merged
- the details of the testing you've done on it (both manual and automated)
- which components are affected by this PR
- links to issues that this PR addresses

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by bcreane** [comment]
@caseydavenport - thanks for fixing up tests. I only checked `test-containerized` but skipped UTs.

File: tests/st/calicoctl/test_crud.py [hunk 1]
```diff
@@ -470,7 +470,7 @@ def test_bgpconfig(self):
 
         # Attempt to delete the default resource by name (i.e. without using a resource version).
         rc = calicoctl("delete bgpconfig %s" % name(rev0))
-        rc.assert_error(DELETE_DEFAULT)
+        rc.assert_no_error()
```
Comment (gunjan5) [inline tests/st/calicoctl/test_crud.py]: How would deleting the default bgpconfig succeed? I thought we don't let anyone delete it from calicoctl, did we change that?

Comment (bcreane) [reply tests/st/calicoctl/test_crud.py]: yeah, just checked and it's now possible to remove the default bgp resource. maybe @robbrockbank has the info.

Comment (gunjan5) [reply tests/st/calicoctl/test_crud.py]: just talked to Rob, we have disabled it for now, but for some tests. I'm not really convinced that's a good reason to disable the check, but for alpha release, it's not necessary and we have a backlog item to rethink this, so I'm not going to block this

**Comment by bcreane** [comment]
Was going to squash, but okay.

**Pull request merged**